implementations/contracts/constants.sol

```diff
@@ -13,5 +17,5 @@ uint256 constant OPERATION_3_STATICCALL = 3;
 uint256 constant OPERATION_4_DELEGATECALL = 4;
 
 // ERC725Y overloaded function selectors
-bytes4 constant SETDATA_SELECTOR = bytes4(keccak256("setData(bytes32,bytes)"));
-bytes4 constant SETDATA_ARRAY_SELECTOR = bytes4(keccak256("setData(bytes32[],bytes[])"));
+bytes4 constant SETDATA_SELECTOR = 0x7f23690c;
```

Comment:
I would suggest to keep the `bytes4(keccak256("setData(bytes32,bytes)"));` here and do the same for ERC725X overloaded selector.

This is more readable and will make it less likely to have error in the function selector (we can spot easily if there is a typo in the function signature being hashed), rather than a `bytes4` literal value.

Reply:
The same reason we put the interfaceId as plain bytes is the same reason why i think we should do it for function selectors.

We can calculate the interfaceId by doing `type(I).interfaceId`, and we can do the method you're suggesting now for selectors, but the main purpose is for easy access and checking the actual content.

Like we have tests for ERC165 Ids we can have test for these selectors. But anws i'm happy to revert the changes back :)